concepticondata/conceptlists/BeijingDaxue-1964-905.tsv

```diff
@@ -581,8 +581,8 @@ BeijingDaxue-1964-905-579	579	挑	290	verb	body-part related actions	三、肢
 BeijingDaxue-1964-905-580	580	措	290	verb	body-part related actions	三、肢體動作	動詞	collect	cuò	7	GATHER	措
 BeijingDaxue-1964-905-581	581	跟	291	verb	body-part related actions	三、肢體動作	動詞	follow	gēn	1085	FOLLOW	跟
 BeijingDaxue-1964-905-582	582	逃跑	291	verb	body-part related actions	三、肢體動作	動詞	flee, run away	táo pǎo	2142	RUN AWAY	逃跑
-BeijingDaxue-1964-905-583	583	脫	292	verb	daily actions	四、日常操作	動詞	take off (clothes)	tuō	1755	UNDRESS	脱
-BeijingDaxue-1964-905-584	584	戴	292	verb	daily actions	四、日常操作	動詞	wear (on head)	dài	976	WEAR	戴
+BeijingDaxue-1964-905-583	583	脫	292	verb	daily actions	四、日常操作	動詞	take off (clothes)	tuō	3538	TAKE OFF	脱
```

Comment:
What is the difference between UNDRESS and TAKE OFF here? Do you think we need it to make more sense of these things? I would say "tuō" is pretty much the same as undress?

Reply:
The reasoning was that, from what I've been told, tuō needs a direct object, so it always refers to taking off particular pieces of clothing, whereas undress is a kind of intransitive taking off.

Reply:
Yes, the question, that I am asking myself here, however, is: does that really reflect the data? The point is: if we find the regular tuō yīfu in other datasets of SEA languages, which has the object and therefore intransifies the verb, as it is usual in Chinese glossing, what is it that they answer in the questionnaire? I would suspect, but this needs to be checked, that the questionnaires even if they explicitly ask for tuō yīfu, they will often only answer tuō or the equivalent in the respective languages under investigation. This should definitely be checked, by looking at cases of tuō in our data so far. Let me do that later.

Reply:
![Screenshot_2019-10-10_12-38-12](https://user-images.githubusercontent.com/2916210/66561969-dd116700-eb5a-11e9-9f93-4e8b4060e7d2.png)

Reply:
Please check the situation here. As far as I can tell, the majority of cases are those where the verb-object compound is used and at least alluded to. 

Here's another account of all verbs that are listed under "undress":

![Screenshot_2019-10-10_12-38-12](https://user-images.githubusercontent.com/2916210/66562084-1ea21200-eb5b-11e9-9f1a-a568ee88738e.png)

You can see that the current linking is in fact very consistent, as the majority of sources are from SEA languages, where Chinese was a glossing language, and that they express "undress", specifically also because Chinese has a specific verb here, while English and German are a bit shakier. I suggest that we leave undress as is here, since the cases are all strikingly similar, and the usage of verb-object or verb-only in Chinese glossing is not consistently interpretable as you would do if you deal with a langauge like German, where verb-object compounds that express intransitive verbs do not exist. 

If we realize in later stages that we should distinguish those, we can do it, but for now it seems to be an overkill, also since the number of cases is not that much, and since I know that the Beijing source is just using a different glossing.

Reply:
However, looking at "take off" in our website, we can find some inconsistencies:

![Screenshot_2019-10-10_12-38-12](https://user-images.githubusercontent.com/2916210/66562358-c9b2cb80-eb5b-11e9-896f-1f4777550743.png)

Dellert should be linked to "undress", and the "take off one's clothes" gloss is obsolete, and should be merged with undress, so the entry in Lama should be linked to undress instead of the longer one we replaced long time ago.

The fact that "take off one's clothes" exists as a gloss is further important, as it shows that "take off" is not needed in any case...

Reply:
I see your point. Some seem to not mean fully undress, though, namely "take off (e.g. clothes)" (Bodt), and also "take off (EX:one's coat )" on Dellert. So I would link those two at "take off" and leave the others with "undress". Would you agree?

I did merge the obsolete gloss already.

Reply:
If that's the only issue, I would merge the pull request and fix this in the next commit afterwards.

Reply:
sorry for late reply: Bodt surely means take of clothes, so I'd leave it with undress, as I want to be able to compare across SEA languages here, and Bodt comes from the area. If Dellert has no real concept that means to undress completely, I'd also leave it there, as it is an overkill to make an extra concept for one concept list, if it is more or less compatible (and you do not know what they actually list there as translations in Northeuralex, or have you checked that?).

Reply:
Okay, I guess you convinced me. I now looked into Northeuralex, and while the words I understand all either mean "take off" or both, it's likely not worth it for a single list. My main motivation was the fact that we do distinguish between the opposites (putting on and getting dressed).

I'll remove "take off", adjust the definition of "undress" to include both meanings, and merge this pull request then.

Reply:
Ah, interesting, I did not know we have the two opposites. This actually points to something important with respect to the relations: To make sure we are getting better in specifying meanings, at least in the Swadesh range and a bit over it, we really need to get back to the relation issue. And this would be an example for tagging something as "opposite". I guess, this may in fact help to make some semantic fields clearer, although it is of course quite difficult.

Reply:
We can only mark them as proper opposites if we distinguish the meanings in both cases, though ;) I don't think we have a hyperonym for PUT ON and DRESS UP.

I intend to systematically add these kinds of concept relations at some point. And yes, they make things clearer, and also help in mapping by showing one what one may be overlooking.

Reply:
Okay, I just submitted a PR that essentially allows to have multiple relations, changing pyconcepticon code minimally. This means, that it should no longer fail, if one changes the order. It was much easier than I thought, to implement this.